src/main.js

```diff
@@ -4,35 +4,32 @@
 
 var coverImg = document.querySelector('.cover-image');
 var title = document.querySelector('.cover-title');
-var descriptor1 = document.querySelector('.tagline-1');
-var descriptor2 = document.querySelector('.tagline-2');
+var tagline1 = document.querySelector('.tagline-1');
+var tagline2 = document.querySelector('.tagline-2');
+var buttonRandomCover = document.querySelector('.random-cover-button');
 
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
-var currentCover;
+var currentCover = new Cover(coverImg, title, tagline1, tagline2);
 
 // Add your event listeners here 👇
-title.addEventListener('load', changeTitle(title));
-coverImg.addEventListener('load', changeCoverImg(coverImg));
-descriptor1.addEventListener('load', changeTagline(descriptor1));
-descriptor2.addEventListener('load', changeTagline(descriptor2));
+window.addEventListener('load', createRandomCover(currentCover));
+buttonRandomCover.addEventListener('click', function() {
+  createRandomCover(currentCover);
+});
 
-// Create your event handlers and other functions here 👇
-
-
-function changeTitle(title) {
-  title.innerText = titles[getRandomIndex(titles)];
-}
 
-function changeCoverImg(coverImg) {
-  coverImg.src = covers[getRandomIndex(covers)];
-}
+// Create your event handlers and other functions here 👇
+function createRandomCover(cover) {
+  cover.title.innerText = titles[getRandomIndex(titles)];
+  cover.cover.src = covers[getRandomIndex(covers)];
+  cover.tagline1.innerText = descriptors[getRandomIndex(descriptors)];
+  cover.tagline2.innerText = descriptors[getRandomIndex(descriptors)];
 
-function changeTagline(descriptor) {
-  descriptor.innerText = descriptors[getRandomIndex(descriptors)];
 }
+//group functions together
```

Comment:
Delete this comment since we implemented already completed what it was reminding us to do.